src/main/java/com/cloudblue/connect/api/models/CBCExtensions.java

```diff
@@ -0,0 +1,56 @@
+package com.cloudblue.connect.api.models;
+
+import com.cloudblue.connect.api.models.common.CBCEntity;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CBCExtensions implements CBCEntity {
+    @JsonProperty
+    private String id;
+    
+    @JsonProperty
+    private String icon;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String status;
```

Comment:
Should be of type Enum with all possible values.

Reply:
Done